Sources/SwiftCrossUI/Views/ForEach.swift

```diff
@@ -6,7 +6,8 @@ public struct ForEachViewChildren<
 
     class Storage {
         var nodes: [ViewGraphNode<Child>] = []
-        var container = GtkSectionBox(orientation: .vertical, spacing: 0).debugName(ForEach<Items, Child>.self)
```

Comment:
I assume this line was too long? Wouldn't it be nicer to fullstop before `.debugName` instead of before the param?

Reply:
Oh yep, that wasn't me that was swift format, it makes some weird decisions sometimes 😅 Maybe I should try fixing that specific one at some point